specs/validator/tableland-openapi-spec.yaml

```diff
@@ -122,52 +122,29 @@ paths:
                 $ref: "#/components/schemas/TxnReceipt"
         "400":
           description: Invalid query/statement value
-  /tables/structure/{hash}:
+  /tables/{chainId}/{tableId}:
```

Comment:
Pretty unfortunate that the diff is shown this way.

Reply:
Does it make sense to have the multipart path here? We're not totally RESTful so it's probably fine, but this sort of implies that a path ending in chainId might be valid? Though I'm not confident in that and maybe you can just ignore me!

Reply:
I guess the proposed style should be fine because is the one we currently use in the SDK.

Reply:
Also, I think you meant `tableId`, not `chainId` in your comment, right? (Just want to make sure I understood your point)

Reply:
No I meant, this: `/tables/{chainId}` implies that that is a valid path that might list tables for a given chain (in the RESTful world). Whereas this: `/tables/{chainId}_{tableId}` might more explicitly indicate that you _need_ both. But I don't have a strong opinion here, so go with your gut.

Reply:
@carsonfarmer, I see, I was doubtful if I was understanding correctly since we defined `/receipt/{chainId}/{txnHash}` in the same way; so I guess your point also applies to that existing one in the new spec?

Reply:
Anyway, I'll keep things as they're. Not really convincing to me to make this ad-hoc decision personally, but we're due in the spec finalization. If it isn't convincing for other people (that probably aren't reading this...), I'd guess would be in the next API version round of discussions.

Thanks for the review!